src/PrestaShopBundle/Form/InvalidConfigurationErrorMessageFactory.php

```diff
@@ -0,0 +1,201 @@
+        $this->langRepository = $langRepository;
+    }
+
+    /**
+     * @param InvalidConfigurationDataErrorCollection $errors
+     * @param FormInterface $form
+     *
+     * @return array
+     *
+     * @throws FieldNotFoundException
+     */
+    public function getErrorMessages(InvalidConfigurationDataErrorCollection $errors, FormInterface $form): array
+    {
+        $messages = [];
+
+        foreach ($errors as $error) {
+            $messages[] = $this->getErrorMessageForConfigurationError($error, $this->getLabel($form, $error->getFieldName()));
+        }
+
+        return $messages;
+    }
+
+    /**
+     * @param InvalidConfigurationDataError $error
+     *
+     * @param string $label
+     *
+     * @return string
+     *
+     */
+    protected function getErrorMessageForConfigurationError(InvalidConfigurationDataError $error, string $label): string
+    {
+        switch ($error->getErrorCode()) {
+            case InvalidConfigurationDataError::ERROR_INVALID_DATE_TO:
+            case InvalidConfigurationDataError::ERROR_INVALID_DATE_FROM:
+                return $this->translator->trans(
+                    'Invalid "%s" date.',
+                    [
+                        $label,
+                    ],
+                    'Admin.Orderscustomers.Notification',
+                );
+            case InvalidConfigurationDataError::ERROR_NO_INVOICES_FOUND:
+                return $this->translator->trans(
+                    'No invoice has been found for this period.',
+                    [],
+                    'Admin.Orderscustomers.Notification'
+                );
+            case InvalidConfigurationDataError::ERROR_NO_ORDER_STATE_SELECTED:
+                return $this->translator->trans(
+                    'You must select at least one order status.',
+                    [],
+                    'Admin.Orderscustomers.Notification'
+                );
+            case InvalidConfigurationDataError::ERROR_NO_INVOICES_FOUND_FOR_STATUS:
+                return $this->translator->trans(
+                    'No invoice has been found for this status.',
+                    [],
+                    'Admin.Orderscustomers.Notification'
+                );
+            case InvalidConfigurationDataError::ERROR_INCORRECT_INVOICE_NUMBER:
+                return $this->translator->trans(
+                    'Invoice number must be greater than the last invoice number, or 0 if you want to keep the current number.',
+                    [],
+                    'Admin.Orderscustomers.Notification'
+                );
+            case InvalidConfigurationDataError::ERROR_CONTAINS_HTML_TAGS:
+                if ($error->getLanguageId()) {
+                    $lang = $this->langRepository->findOneBy(['id' => $error->getLanguageId()]);
+
+                    return $this->translator->trans(
+                        'The "%s" field in "%s" is invalid. Field must not contain HTML tags.',
```

Comment:
```suggestion
                        'The "%s" field in %s is invalid. Field must not contain HTML tags.',
```

Following your comment in another PR, I don't think we need quotation marks for the language here. :)